src/Console/Commands/PermissionSeedCommand.php

```diff
@@ -45,7 +44,13 @@ public function handle()
 
         config(['translatable.translation_models_path' => 'Mabrouk\Permission\Models']);
 
-        $this->call('db:seed', ['--class' => RoleableSeeder::class]);
+        $seedOptions = ['--class' => RoleableSeeder::class];
+
+        if (config('translatable.run_seed_in_production')) {
```

Comment:
**Blocking**

we should replace "translatable" by "permissions".

**Important**

we may change the config key name to be "force_seeding_without_questions" with default value "true"